synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp

```diff
@@ -195,25 +195,21 @@ ffmpeg_trgt::init(ProgressCallback *cb=NULL)
 	}
 
 	String ffmpeg_binary_path;
-	std::list< String > binary_choices;
-	binary_choices.push_back("ffmpeg");
-	binary_choices.push_back("avconv");
-	std::list< String >::iterator iter;
-	for(iter=binary_choices.begin();iter!=binary_choices.end();iter++)
-	{
+	std::list<std::string> binary_choices = {"ffmpeg", "avconv"};
+	for(const auto& iter : binary_choices) {
```

Comment:
It isn't an iterator but the element "itself", so I think `iter` isn't a good variable name.

Reply:
Ok